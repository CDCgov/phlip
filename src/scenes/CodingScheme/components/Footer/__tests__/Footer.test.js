import React from 'react'
import { shallow } from 'enzyme'
import { Footer } from '../index'

describe('CodingScheme -- Footer component', () => {
  test('should render correctly', () => {
    expect(shallow(<Footer />)).toMatchSnapshot()
  })
})
