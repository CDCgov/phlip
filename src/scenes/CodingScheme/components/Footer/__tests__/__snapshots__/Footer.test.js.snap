// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodingScheme -- Footer component should render correctly 1`] = `
<Row
  component={<WithStyles />}
  displayFlex={true}
  elevation={4}
  flex={false}
  reverse={true}
  style={
    Object {
      "alignItems": "center",
      "height": 64,
      "padding": "0 24px",
    }
  }
>
  <Link
    replace={false}
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/"
  >
    <WithTheme
      closeButton={true}
      value="Close"
    >
      Close
    </WithTheme>
  </Link>
</Row>
`;
